### PR TITLE
Update lang names in `config.toml`

### PR DESCRIPTION
**Pull request by jihoon-seo**
### Describe your changes

- Add space between each language name in their own language (e.g., `한국어`) and its corresponding English name (e.g., `(Korean)`)
  - AFAIK maintainers agreed to do so someday ago..?
- `中文(Chinese)` → `简体中文 (Simplified Chinese)`
  - To adopt 'Traditional Chinese' alongside in near future

### Related issue number or link (ex: `resolves #issue-number`)
None

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -91,28 +91,28 @@ weight = 4
...
-languageName ="Español(Spanish)"
+languageName ="Español (Spanish)"
 contentDir = "content/es"
 weight = 6
 
 [languages.zh-cn]
 title = "Cloud Native(云原生) Glossary"
 description = "CNCF 云原生 Glossary 项目旨在用作讨论云原生应用程序时常用术语的参考"
-languageName ="中文(Chinese)"
+languageName ="简体中文 (Simplified Chinese)"
```
Comment (jihoon-seo) [inline config.toml:108]: [Language name candidates in Chinese characters]
1. `简化字` / `繁体字`
2. `简体中文` / `繁體中文`
3. `中文简体` / `中文繁體`
4. Something else

[Language name candidates in English]
1. `Simplified Chinese` / `Traditional Chinese`
2. `Chinese Simplified` / `Chinese Traditional`
3. Something else

@hanyuancheung @Jacob953 @Rocksnake @Submarinee
@pichuang @ydFu
Do you guys have any preference or suggestion among the options above?

(FYI: These strings are to be displayed in the language selector of the Glossary website.)
image

**Comment by pichuang** [comment]
In my personal opinion, 

[Language name candidates in Chinese characters]
2. 简体中文 / 繁體中文

[Language name candidates in English]
1. Simplified Chinese / Traditional Chinese

**Comment by seokho-son** [comment]
Hi @jihoon-seo 
I guess you read this related thread already. https://cloud-native.slack.com/archives/C02N2RGFXDF/p1682585391111919?thread_ts=1682521759.437899&cid=C02N2RGFXDF

簡体中文 (Simplified Chinese), 繁體中文 (Traditional Chinese), and 粵語 (Cantonese)

**Review by seokho-son — APPROVED** [review]
LGTM! :)

**Comment by ydFu** [comment]
my personal suggestion：

[Language name candidates in Chinese characters]
2. 简体中文 / 繁體中文

[Language name candidates in English]

Simplified Chinese / Traditional Chinese Language name candidates in Chinese characters]

2. 简体中文 / 繁體中文

[Language name candidates in English]

1. Simplified Chinese / Traditional Chinese

**Comment by Rocksnake** [comment]
In my personal opinion, 

[Language name candidates in Chinese characters]：简体中文 / 繁體中文.
[Language name candidates in English]: Simplified Chinese / Traditional Chinese.

I personally think that these descriptions is more appropriate.

**Review by CathPag — APPROVED** [review]
LGTM! Glad to see the added spacing 🎉

**Pull request merged**